build_library/grub.cfg

```diff
@@ -27,6 +27,32 @@ if [ "$grub_platform" = "efi" ]; then
    getenv -e SetupMode -g 8be4df61-93ca-11d2-aa0d-00e098032b8c -b setupmode
    if [ "$sb" = "01" -a "$setupmode" = "00" ]; then
       set secure_boot="1"
+      getenv -e gpgpubkey -g b8ade7d5-d400-4213-8d15-d47be0a621bf -b gpgpubkey
```

Comment:
I meant descriptive name for what it is used for, GrubVerificationKey, or NetworkBootVerificationKey or something.